### ISSUE TITLE
s3 relies on outdated mime package with security issue

### ISSUE THREAD

**Issue by naderm** [andrewrk/node-s3-client#190]
It looks like node-s3-client requires mime@~1.2.11 which is vulnerable to a [regular expression denial of service exploit](https://nodesecurity.io/advisories/535). This exploit is fixed in mime@^1.4.1 or mime@^2.0.3

**Comment by carterbancroft** [comment]
Yes, this is breaking our builds. I've submitted a PR to bump that version here https://github.com/andrewrk/node-s3-client/pull/191

Can we merge this?

**Comment by matrus2** [comment]
+1

**Comment by matrus2** [comment]
This repository seems to be dead. I am going to either change it to something else or fork it. Last commit was in Jan 19, 2017.

**Comment by matrus2** [comment]
FYI: Fork with updated dependencies:

https://github.com/matrus2/node-s3-client

**Comment by breathe** [comment]
Thank you @matrus2 -- your fork works for me (appears to resolve an unrelated bug I was hitting)!

Recommend -- are you planning to maintain the fork?

**Comment by matrus2** [comment]
@breathe Yes, this is a plan.
